wagtailio/core/models/snippets.py

```diff
@@ -0,0 +1,73 @@
+from django.db import models
+
+from wagtail.admin.edit_handlers import FieldPanel, MultiFieldPanel, StreamFieldPanel
+from wagtail.core.fields import StreamField
+from wagtail.snippets.models import register_snippet
+
+from wagtailio.core.blocks import GetStartedBlock
+
+
+@register_snippet
+class GetStartedSnippet(models.Model):
+    name = models.CharField(max_length=255)
+    body = StreamField(
+        [
+            (
+                "get_started_block",
+                GetStartedBlock(),
+            ),
+        ],
+        max_num=1,
+    )
+
+    panels = [
+        FieldPanel("name"),
+        StreamFieldPanel("body"),
+    ]
+
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        verbose_name = "Get started block"
+
+
+@register_snippet
+class SignupFormSnippet(models.Model):
+    name = models.CharField(
+        max_length=255,
+        help_text="For internal identification e.g 'This Week in Wagtail', 'TWIW'.",
+    )
+    heading = models.CharField(max_length=255)
+    sub_heading = models.TextField(blank=True)
+    mailchimp_account_id = models.CharField(
```

Comment:
Question: Is this going to be the same account every time? It feels like it could be more appropriate in some settings / env vars.